EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc

```diff
@@ -266,9 +268,12 @@ void GEMDigiToRawModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
     fedRawData.resize(dataSize);
 
     uint64_t* w = reinterpret_cast<uint64_t*>(fedRawData.data());
-    for (const auto& word : words)
+    for (const auto& word : words) {
+#ifdef EDM_ML_DEBUG
```

Comment:
It is curious that you use both `LogDebug` and `#ifdef EDM_ML_DEBUG `in the same class: was any of them a remnant of your validation of the fix, or are they supposed to remain both?
In any case, a 
```C++
#include "FWCore/MessageLogger/interface/MessageLogger.h"
```
statement is missing in both plugins

Reply:
@perrotta Only LogDebug was used originally.  I added a few more LogDebug and used EDM_ML_DEBUG for the output to be easier to see. I can change them all to either one if that is preferred.

Reply:
I think that having one single method to report debug logs can be less confusing, and the one stated in the MessageLogger are typically recommendedin CMSSW instead of cout's (which aren't thread safe, in case you want to use them in a multithread environment). I won't be strict on it, though, and I just wanted to point it out to you.
In any case, if you stick on `LogDebug,` please add the missing header includes

Reply:
done